ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/python
```

Comment:
Acceptable, after all the changes I will make a pull request on framework-alba-plugin repository